app/controllers/notifications_controller.rb

```diff
@@ -1,20 +1,25 @@
 class NotificationsController < ApplicationController
   before_filter :if_not_signed_in
-  before_action :set_notification, only: [:destroy]
```

Comment:
Cool, glad someone else noticed it earlier!  I'll sync up and work my changes around theirs.  Thanks for the heads up.